templates/gift_card.liquid

```diff
@@ -118,13 +120,22 @@
         {%- assign gift_card_balance = gift_card.balance | money -%}
       {% endif %}
       {%- if gift_card.balance != gift_card.initial_value -%}
-        <p class="gift-card__label caption-large">{{ 'gift_cards.issued.remaining_html' | t: balance: gift_card_balance }}</p>
+        <p class="gift-card__label caption-large">
+          {{ 'gift_cards.issued.remaining_html' | t: balance: gift_card_balance }}
+        </p>
```

Comment:
I fee like here there is a need for `-}}` here too no ?

Reply:
No because `p` tags are `display: block`. Which means they are not inner leading whitespace sensitive or inner trailing whitespace sensitive.